libs/fairness/src/lib/FairnessWizard.tsx

```diff
@@ -175,6 +191,12 @@ export class FairnessWizardV2 extends React.PureComponent<
       defaultFairnessMetricPrioritization
     );
 
+    if (this.props.errorKey === "disabled") {
+      selectedErrorKey = "disabled";
+    } else {
+      selectedErrorKey = "enabled";
```

Comment:
So if we run this with precomputed data and don't explicitly mark "disabled", will it assume that it's enabled?

Reply:
Yes I think the only case you wouldn't want them to be enabled would be if it's legacy data (in which you would explicitly disabled them)